### ISSUE TITLE
Improvement in UI

### ISSUE THREAD

**Issue by thinktocode** [thinktocode/GitFit#1]
Greetings!
The UI of the project is not so good. It can be improved a lot.

**Comment by himanshu007-creator** [comment]
i would like to workon it

**Comment by thinktocode** [comment]
Good Luck!

**Comment by himanshu007-creator** [comment]
Can you elaborate more about your requirements?

**Comment by thinktocode** [comment]
You might have downloaded the app, the UI is not so good. So you need to figure out where UI needs to be updated. Thats what I can tell you. Good Luck

**Comment by himanshu007-creator** [comment]
i have to work on code base or suggest some improvements via images alone?
i  am actually comfortable in web development but have never developed an app.

**Comment by thinktocode** [comment]
Bro, I think you should not work on it then. I'm removing you from the assignee. Because you are not good at it. You can check some web projects under DWOC.

**Comment by himanshu007-creator** [comment]
> Bro, I think you should not work on it then. I'm removing you from the assignee. Because you are not good at it. You can check some web projects under DWOC.

sure!

**Comment by kanishkaa24** [comment]
Please assign me this issue.

**Comment by kanishkaa24** [comment]
https://user-images.githubusercontent.com/71253790/104470685-fc116b00-55df-11eb-98a5-b23de4c28757.mp4
I have made one enhancement. I am working on the others. Please tell me if you like this.

**Comment by thinktocode** [comment]
Its a good one but the issue meant about the UI of the app and I think this is just a splash screen.

**Comment by kanishkaa24** [comment]
> Its a good one but the issue meant about the UI of the app and I think this is just a splash screen.

Yes, I am working on that.

**Comment by thinktocode** [comment]
I'm sorry what do you mean by Design?

**Comment by kanishkaa24** [comment]
I mean designing the interface, not coding it.

**Comment by thinktocode** [comment]
You cannot be assigned the issue then. This is a normal open source project. Do you think that there are different people for designing and coding?
Thank you
Good luck

**Comment by kanishkaa24** [comment]
> You cannot be assigned the issue then. This is a normal open source project. Do you think that there are different people for designing and coding?
> Thank you
> Good luck

I'm sorry but there are.

**Comment by thinktocode** [comment]
Yes, I know that there are, but not in small OPEN SOURCE PROJECTS. This is not a project of a huge company.

**Comment by kanishkaa24** [comment]
The projects included in this particular event contain separate issues for designing and separate issues for coding. You can checkout them.

**Comment by thinktocode** [comment]
I don't need to check anyone else's project. I can just say that my project does not do that.

**Comment by Dhriti-Roy** [comment]
Can I work on this issue.

**Comment by Dhriti-Roy** [comment]
Kindly assign this to me...

**Comment by Dhriti-Roy** [comment]
Thankyou for assigning me forthis issue....I'll be submitting the PR soon
